tests/test_smile.py

```diff
@@ -473,22 +473,24 @@ async def tinker_thermostat_temp(self, smile, loc_id, unhappy=False):
         """Toggle temperature to test functionality."""
         _LOGGER.info("Asserting modifying settings in location (%s):", loc_id)
         tinker_temp_passed = False
-        for new_temp in [20.0, 22.9]:
-            _LOGGER.info("- Adjusting temperature to %s", new_temp)
-            try:
-                await smile.set_temperature(loc_id, new_temp)
-                tinker_temp_passed = True
-                _LOGGER.info("  + worked as intended")
-            except (
-                pw_exceptions.ErrorSendingCommandError,
-                pw_exceptions.ResponseError,
-            ):
-                tinker_temp_passed = False
-                if unhappy:
-                    _LOGGER.info("  + failed as expected")
-                else:  # pragma: no cover
-                    _LOGGER.info("  - failed unexpectedly")
-                    raise self.UnexpectedError
+        test_temp = {"setpoint": 22.9}
+        if smile._anna_cooling_present:
+            test_temp = {"setpoint_low": 19.5, "setpoint_high": 23.5}
```

Comment:
easier to just log and then return True? same with the except?

Reply:
Implemented as suggested.